src/routing/wrappers/Probe.svelte

```diff
@@ -34,6 +34,14 @@
   });
 </script>
 
+<svelte:head>
+  {#if $store.probeName}
+    <title>GLAM: {$store.probeName}</title>
```

Comment:
What are your thoughts on reversing this order? That is, something like:

```svelte
    <title>{$store.probeName} - GLAM</title>
```

or

```svelte
    <title>{$store.probeName} | GLAM</title>
```

It's a common pattern, maybe more common than what we have now, and I think it would be especially helpful in GLAM where the name of the probe is so important. I think the probe name plus the favicon should give the user all the context they need.

Reply:
I like this suggestion @openjck. Also my tabs tend to get short so having the probe name is the best thing while scanning, and the favicon does the work of saying what thing it is